src/pages/internal/test-ui.tsx

```diff
@@ -0,0 +1,24 @@
+import { Calendar, Icon, Mail } from "lucide-react";
+import GithubAuthButton from "~/components/auth/githubauth-button";
+import GoogleAuthButton from "~/components/auth/googleauth-button";
+import { Button } from "~/components/ui/button";
+import { Input } from "~/components/ui/input";
+import { Textarea } from "~/components/ui/textarea";
+
+function TestUI() {
+  return (
+    <div className="flex h-screen items-center justify-center">
+      <div className="flex w-[30%] flex-col items-center justify-center gap-3">
+        <GithubAuthButton redirect="/" />
+        <GoogleAuthButton redirect="/" />
+        <Button variant="primary"> Continue</Button>
+        <Button variant="outline"> Continue</Button>
+        <Button variant="destructive">Delete</Button>
+        <Input placeholder="john@doe.com" variant="primary" />
+        <Textarea placeholder="Enter answer here" variant="primary" />
+      </div>
+    </div>
+  );
+}
```

Comment:
I just realized we already have a ui page, can we used `/pages/ui` instead (it doesn't matter that this page is public because it's just ui components LOL)

Reply:
actually I don't care enough about this LOL